carla/__init__.py

```diff
@@ -1,5 +1,21 @@
 # flake8: noqa
+# isort:skip
+import logging
 
+from ._logger import INFOFORMATTER
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+# defines the stream handler
+_ch = logging.StreamHandler()
+_ch.setLevel(logging.INFO)
```

Comment:
Not really sure, I understand the logging configuration here, are we configuring the DEBUG level or the INFO level?

Reply:
Should be INFO, sorry 😅 
But made no difference in my tests.